libsel4/include/api/syscall.xml

```diff
@@ -50,12 +50,7 @@
             <syscall name="DebugNameThread"/>
         </config>
         <config>
-            <condition>
-                <and>
-                    <config var="CONFIG_DEBUG_BUILD"/>
-                    <config var="CONFIG_ENABLE_SMP_SUPPORT"/>
-                </and>
-            </condition>
+            <condition><config var="CONFIG_DEBUG_BUILD"/></condition>
             <syscall name="DebugSendIPI"/>
         </config>
```

Comment:
@kent-mcleod was this intentional? It doesn't look to me like `DebugSendIPI` should be available in multikernel configs.

Reply:
It was intentional but it's not a required change. Mainly because the CONFIG_ENABLE_SMP_SUPPORT was previously required because the implementation was behind the CONFIG_ENABLE_SMP_SUPPORT flag.  Because the implementation moved out of the CONFIG_ENABLE_SMP_SUPPORT flag, there was no reason for preventing it at the config level.

Reply:
Ok, makes sense. I guess it does not hurt and it's debug functionality anyway. Happy to leave it as is.